crates/router/src/routes/routing.rs

```diff
@@ -111,6 +159,23 @@ pub async fn routing_retrieve_config(
     .await
 }
 
+#[utoipa::path(
+    get,
+    path = "/routing",
+    params(
+        ("limit" = Option<u16>, Query, description = "The number of records to be returned"),
+        ("offset" = Option<u8>, Query, description = "The record offset from which to start gathering of results"),
+        ("profile_id" = Option<String>, Query, description = "The unique identifier for a merchant profile"),
+    ),
+    responses(
+        (status = 200, description = "Successfully fetched routing dictionary", body = RoutingKind),
+        (status = 500, description = "Internal server error"),
+        (status = 404, description = "Resource missing")
+    ),
+   tag = "Routing",
+   operation_id = "Retrieve routing dictionary",
```

Comment:
Rename this route to `list_routing_configs` and remove the word `dictionary` from the docs.